client/src/components/Task1/Task1.js

```diff
@@ -34,7 +38,13 @@ function Task1() {
                     <Item>Component Here</Item>
                 </Grid>
                 <Grid xs={6}>
-                    <Item>Component Here</Item>
+                    <Item>
+                        <Grid container>
+                            {/* Display Processed QR Data */}
+                            <Grid item xs={6}><QRData qrType="1" qrReady={qrReady === "1"}/></Grid>
```

Comment:
I think qrReady is an int so should be qrReady === 1

Reply:
Removed equality check